ChangeLog

```diff
@@ -1,3 +1,10 @@
+Release 0.2.11 - 2017-04-23
+
+* Set http_proxy.https option to true by default v2 [#54]
+* Add http_proxy option to README.md [#51]
+* Add HttpProxy option [#48]
```

Comment:
Yea, not easy to read. Added 8c7527f to group several PRs #48,#51,#54. How about this?